components/plan-build/bin/hab-plan-build.sh

```diff
@@ -1229,8 +1233,8 @@ _print_recursive_deps() {
   local dep
   # Compute the amount of leading whitespace when display this line and any
   # child dependencies.
-  local padn=$(($level * 4))
-  while read dep; do
+  local padn=$((level * 4))
```

Comment:
See https://github.com/koalaman/shellcheck/wiki/SC2004#-is-unnecessary-on-arithmetic-variables